### ISSUE TITLE
Submit report to zoom regarding missing acccessibility features

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#16]
Zoom is not easily accessible for remote interviews.  Zoom claims to be committed to accessibility (https://zoom.us/accessibility) and should be notified why they are inaccessible for our needs.

Should they investigate our report, we might have a way to conduct remote interviews without relying on PhET-iO, which has proven to be challenging (see https://github.com/phetsims/a11y-research/issues/2#issuecomment-270181542)

@emily-phet @terracoda could you please write in this issue what the problems are with Zoom that make it inaccessible?  We can then submit a report to them.

**Comment by emily-phet** [comment]
@jessegreenberg - our experience with Zoom has been that blind students and staff on campus have difficulty using it, and prefer to use the mobile version over the desktop version. We have not asked someone using a screen reader to join a call and screen share from their device to determine specifically what issues the users run into. We can do this, and determine if the screen share option is accessible, and what steps are needed to get there. Additionally, I have also asked many screen reader users if there is a conference call application that they find more easy to use, and the response has been that all of them are difficult. 

Ultimately, I think this is an issue of technical versus functional accessibility. Zoom is probably technically accessible, but in practice it is not (for at least some users) functionally accessible. This ambiguity in functional accessibility is one of the challenges we had hoped to overcome with more fine-tuned data collection tools, but if that is not the case we'll just have to map out common access issues and walk users through them prior to data collection.

**Comment by jessegreenberg** [comment]
I see, thanks for clarifying @emily-phet.

> We can do this, and determine if the screen share option is accessible, and what steps are needed to get there.

Perhaps if we do this we can have a more detailed report to send along to Zoom.  It looks like they make it very easy to send feedback:
> Zoom has made ongoing efforts to monitor and improve our accessibility features, and has resources dedicated to accessibility questions and concerns. To reach us, please email access@zoom.us or open a support ticket at support.zoom.us.

**Comment by terracoda** [comment]
@jessegreenberg, @emily-phet, is the plan to check it ourselves first in VO, JAWS and NVDA and then ask for assistance from a screen reader user?

**Comment by jessegreenberg** [comment]
Yes, that sounds like a good way to get started @terracoda.  I can try it out on all three today to see if I can find any features that are blatantly not accessible, but I suppose we won't really know for sure until we get assistance from a screen reader user.

**Comment by jessegreenberg** [comment]
However, I see no mention of JAWS at https://zoom.us/accessibility

> Zoom services are now compatible with standard screen readers such as VoiceOver on iOS and OSX platforms, TalkBack on Android devices, and NVDA for Windows platforms

**Comment by jessegreenberg** [comment]
My experience with NVDA:
- Tab navigation works for the toolbar at the bottom, but only while it is visible.  Without making the toolbar appear with my mouse (which I already knew how to do) I can't access any of the options for video sharing.
- When I have tab focus on a button in the bottom navigation toolbar, the toolbar still dissapears after ~3 seconds, and I can no longer focus anything other than the video share window.
- If I open "Share screen" in time, the "Share Window" dialog is nicely accessible, and I can choose which window to share. 
- If I enter a window that is different than the one I chose to share, screen sharing pauses.
- Once I share my screen, I can't focus on the zoom toolbar again without clicking on it with a mouse. Zoom does not appear in list of open apps that I can alt+tab focus.

**Comment by jessegreenberg** [comment]
I wonder if I am missing something some commands here, this is NVDA outside of web content, which I am not really used to.

**Comment by jessegreenberg** [comment]
Zoom has special hotkeys: https://support.zoom.us/hc/en-us/articles/205683899-Hot-Keys-for-Zoom

**Comment by jessegreenberg** [comment]
Cool, lots of features in https://support.zoom.us/hc/en-us/articles/205683899-Hot-Keys-for-Zoom.  ctrl + alt+shift fixes all concerns in https://github.com/phetsims/a11y-research/issues/16#issuecomment-271355106.  It will focus toolbar when sharing screen as well.

**Comment by jessegreenberg** [comment]
JAWS works very well with Zoom's custom hotkeys.

**Comment by jessegreenberg** [comment]
VoiceOver seems to work very well too, though I had to use the hotkeys listed in the docuement.  One minor issue was that the "Share Screen" button was listed as a 'checked checkbox' when I wasn't sharing screens, but I don't think that would really inhibit use too much.

@terracoda @emily-phet Zoom seems pretty accessible to me, as long as the user has access to these hotkeys: https://support.zoom.us/hc/en-us/articles/205683899-Hot-Keys-for-Zoom. Of course, a native screen reader user might have a totally different experience or expectations.

**Comment by terracoda** [comment]
@jessegreenberg, thanks for the documentation on this. This is great.

One other issue that may be a problem during remote recordings is that the conference software automatically adjusts the volume of the screen reader output when the user is speaking. This fluctuation in volume interferes with a steady recording volume. I have not tested this with Zoom, but this seems to happens with Skype.

**Comment by emily-phet** [comment]
@terracoda Good point! I ran into this with Skype also. I think there's a plugin for Skype that helps with this issue, not sure if it will be a problem for Zoom or a way around it. Ashley will be joining our Tuesday meeting and I'll ask her if we can do a quick test.

**Comment by emily-phet** [comment]
Closing, as we run into specific issues with Zoom and remote user testing we can submit bug reports and track in a11y-research repo.
